### PR TITLE
feat(lib/Utils/StringHelpers.php): Added DocBlocks

### PR DESCRIPTION
**Pull request by vandebled**

### CODE REVIEW

File: lib/Utils/StringHelpers.php [hunk 1]
```diff
@@ -4,11 +4,27 @@
...
     }
 
+    /**
+    * Strip all HTML tags including script and style,
+    * and trims text to a certain number of words.
+    *
+    * @since 0.1.0
+    *
+    * @param string $str    The string to trim and strip.
+    * @param number $length The string lenght to return.
```
Comment (Qakulukiam) [inline lib/Utils/StringHelpers.php]: typo: length

File: lib/Utils/StringHelpers.php [hunk 2]
```diff
@@ -4,11 +4,27 @@
 
 class StringHelpers
 {
+    /**
+    * Converts a string from Camel case to Kebap case.
```
Comment (Qakulukiam) [inline lib/Utils/StringHelpers.php]: camel and kebap should probably be lower case

File: lib/Utils/StringHelpers.php [hunk 3]
```diff
@@ -4,11 +4,27 @@
...
+    *
+    * @since 0.1.0
+    *
+    * @param string $str  The string to convert.
```
Comment (Qakulukiam) [inline lib/Utils/StringHelpers.php]: extra space between variable name and description

File: lib/Utils/StringHelpers.php [hunk 4]
```diff
@@ -4,11 +4,27 @@
...
+    */
     public static function camelCaseToKebap($str)
     {
         return strtolower(preg_replace('/([a-zA-Z])(?=[A-Z])/', '$1-', $str));
     }
 
+    /**
+    * Strip all HTML tags including script and style,
```
Comment (Qakulukiam) [inline lib/Utils/StringHelpers.php]: Strips

File: lib/Utils/StringHelpers.php [hunk 5]
```diff
@@ -17,6 +33,13 @@ public static function trimStrip($str, $length = 25)
         return $str;
     }
 
+    /**
+    * Splits a Camel case string.
```
Comment (Qakulukiam) [inline lib/Utils/StringHelpers.php]: camel

File: lib/Utils/StringHelpers.php [hunk 6]
```diff
@@ -17,6 +33,13 @@ public static function trimStrip($str, $length = 25)
...
+    *
+    * @since 0.1.0
+    *
+    * @param string $str  The string to split.
```
Comment (Qakulukiam) [inline lib/Utils/StringHelpers.php]: extra space

File: lib/Utils/StringHelpers.php [hunk 7]
```diff
@@ -29,6 +52,14 @@ public static function splitCamelCase($str)
         return implode($a, ' ');
     }
 
+    /**
+    * Converts a string from Kebap case to Camel case.
```
Comment (Qakulukiam) [inline lib/Utils/StringHelpers.php]: kebap and camel

File: lib/Utils/StringHelpers.php [hunk 8]
```diff
@@ -46,11 +85,31 @@ public static function removePrefix($prefix, $str)
         return $str;
     }
 
+    /**
+    * Checks if a string starts with a certain string.
+    *
+    * @since 0.1.0
+    *
+    * @param string $search   The string to search.
```
Comment (Qakulukiam) [inline lib/Utils/StringHelpers.php]: This sounded confusing in my head, maybe go with "The string to search for."

File: lib/Utils/StringHelpers.php [hunk 9]
```diff
@@ -46,11 +85,31 @@ public static function removePrefix($prefix, $str)
...
     {
         return substr($subject, 0, strlen($search)) === $search;
     }
 
+    /**
+    * Checks if a string ends with a certain string.
+    *
+    * @since 0.1.0
+    *
+    * @param string $search   The string to search.
```
Comment (Qakulukiam) [inline lib/Utils/StringHelpers.php]: same here

File: lib/Utils/StringHelpers.php [hunk 10]
```diff
@@ -38,6 +69,14 @@ public static function kebapCaseToCamelCase($str, $capitalizeFirstCharacter = fa
         return $str;
     }
 
+    /**
+    * Removes a prefix from a string.
+    *
+    * @since 0.1.0
+    *
+    * @param string $prefix The prefix to be removed.
+    * @param string $str    The string to manipulate.
```
Comment (Qakulukiam) [inline lib/Utils/StringHelpers.php]: missing `@return`

File: lib/Utils/StringHelpers.php [hunk 11]
```diff
@@ -29,6 +52,14 @@ public static function splitCamelCase($str)
         return implode($a, ' ');
     }
 
+    /**
+    * Converts a string from Kebap case to Camel case.
+    *
+    * @since 0.1.0
+    *
+    * @param string $str                        The string to convert.
+    * @param boolean $capitalizeFirstCharacter  Sets if the first character should be capitalized.
```
Comment (Qakulukiam) [inline lib/Utils/StringHelpers.php]: missing `@return`

File: lib/Utils/StringHelpers.php [hunk 12]
```diff
@@ -17,6 +33,13 @@ public static function trimStrip($str, $length = 25)
         return $str;
     }
 
+    /**
+    * Splits a Camel case string.
+    *
+    * @since 0.1.0
+    *
+    * @param string $str  The string to split.
+    */
```
Comment (Qakulukiam) [inline lib/Utils/StringHelpers.php]: missing `@return`

File: lib/Utils/StringHelpers.php [hunk 13]
```diff
@@ -4,11 +4,27 @@
...
 
+    /**
+    * Strip all HTML tags including script and style,
+    * and trims text to a certain number of words.
+    *
+    * @since 0.1.0
+    *
+    * @param string $str    The string to trim and strip.
+    * @param number $length The string lenght to return.
+    */
```
Comment (Qakulukiam) [inline lib/Utils/StringHelpers.php]: missing `@return`

File: lib/Utils/StringHelpers.php [hunk 14]
```diff
@@ -4,11 +4,27 @@
 
 class StringHelpers
 {
+    /**
+    * Converts a string from Camel case to Kebap case.
+    *
+    * @since 0.1.0
+    *
+    * @param string $str  The string to convert.
+    */
```
Comment (Qakulukiam) [inline lib/Utils/StringHelpers.php]: missing `@return`

File: lib/Utils/StringHelpers.php (hunk 2 above)
Comment (vandebled) [inline lib/Utils/StringHelpers.php]: ok

File: lib/Utils/StringHelpers.php (hunk 3 above)
Comment (vandebled) [inline lib/Utils/StringHelpers.php]: removed

File: lib/Utils/StringHelpers.php (hunk 14 above)
Comment (vandebled) [inline lib/Utils/StringHelpers.php]: added

File: lib/Utils/StringHelpers.php (hunk 4 above)
Comment (vandebled) [inline lib/Utils/StringHelpers.php]: changed

File: lib/Utils/StringHelpers.php (hunk 1 above)
Comment (vandebled) [inline lib/Utils/StringHelpers.php]: changed

File: lib/Utils/StringHelpers.php (hunk 13 above)
Comment (vandebled) [inline lib/Utils/StringHelpers.php]: added

File: lib/Utils/StringHelpers.php (hunk 5 above)
Comment (vandebled) [inline lib/Utils/StringHelpers.php]: done

File: lib/Utils/StringHelpers.php (hunk 6 above)
Comment (vandebled) [inline lib/Utils/StringHelpers.php]: removed

File: lib/Utils/StringHelpers.php (hunk 12 above)
Comment (vandebled) [inline lib/Utils/StringHelpers.php]: added

File: lib/Utils/StringHelpers.php (hunk 7 above)
Comment (vandebled) [inline lib/Utils/StringHelpers.php]: ok

File: lib/Utils/StringHelpers.php (hunk 11 above)
Comment (vandebled) [inline lib/Utils/StringHelpers.php]: added

File: lib/Utils/StringHelpers.php (hunk 10 above)
Comment (vandebled) [inline lib/Utils/StringHelpers.php]: added

File: lib/Utils/StringHelpers.php (hunk 8 above)
Comment (vandebled) [inline lib/Utils/StringHelpers.php]: changed

File: lib/Utils/StringHelpers.php (hunk 9 above)
Comment (vandebled) [inline lib/Utils/StringHelpers.php]: oki

**Pull request merged**